extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -99,14 +99,16 @@ export const Selectors = {
   ALL_AMP_MEDIA:
     'amp-story-grid-layer amp-audio, ' +
     'amp-story-grid-layer amp-video, amp-story-grid-layer amp-img, ' +
-    'amp-story-grid-layer amp-anim',
+    'amp-story-grid-layer amp-anim, ' +
+    'amp-story-grid-layer amp-story-360',
```

Comment:
Since amp-story-360 has an underlying amp-img or amp-video, I think we don't need to add it specifically here. It'll already wait for the underlying element to load.

Reply:
Ack.